cats-backend/src/models/cat.model.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  Table,
+  Column,
+  Model,
+  PrimaryKey,
+  AutoIncrement,
+  DataType,
+  HasMany,
+} from 'sequelize-typescript';
+import { Mouse } from './mouse.model';
+
+@Table
+export class Cat extends Model<Cat> {
+  @PrimaryKey
+  @AutoIncrement
+  @Column(DataType.INTEGER)
+  declare id: number;
+
+  @Column(DataType.STRING)
+  declare firstName: string;
+
+  @Column(DataType.STRING)
+  declare lastName: string;
+
+  @Column(DataType.STRING)
+  declare image: string;
+
+  @Column(DataType.STRING)
+  declare description: string;
+
+  @HasMany(() => Mouse)
```

Comment:
```suggestion
  @HasMany(() => Mouse, { onDelete: 'CASCADE' })
```

 You should check that if we delete the cat, it deletes the mice that the cat "holds" with CASCADE - Cascade means to delete the unused records (mice in our case)

Reply:
i changed it in the previous commit